libpoolprotocols/CMakeLists.txt

```diff
@@ -11,5 +11,5 @@ hunter_add_package(OpenSSL)
 find_package(OpenSSL REQUIRED)
 
 add_library(poolprotocols ${SOURCES})
-target_link_libraries(poolprotocols PRIVATE devcore ethminer-buildinfo ethash::ethash Boost::system jsoncpp_lib_static OpenSSL::SSL OpenSSL::Crypto)
+target_link_libraries(poolprotocols PRIVATE devcore ethminer-buildinfo ethash::ethash Boost::system jsoncpp_static OpenSSL::SSL OpenSSL::Crypto)
```

Comment:
Our linker can't find jsoncpp anymore now. Please mention changes like these in a changelog, readme or similar.

Reply:
updated in e55fbdc2f985b852de61c1b541eb1570ad69dbf0